tests/simple-test.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { test, expect } from '@playwright/test';
+
+test('hello team text', async ({ page }) => {
+  await page.goto('http://localhost:3000');
```

Comment:
❌ 
instead of each time writing the full URL you can add baseUrl to the configurations of playwright test, after that we will be able to use relative paths [see docs](https://playwright.dev/docs/test-configuration)
```ts
await page.goto('/');
```